### PR TITLE
Added build step to verify agent OS

### PR DESCRIPTION
**Pull request by chidozieononiwu**
@Azure/azure-sdk-eng

### CODE REVIEW

**Comment by adxsdk6** [comment]
Can one of the admins verify this patch?

**Comment by adxsdk6** [comment]
Can one of the admins verify this patch?

File: .azure-pipelines/client.yml [hunk 1]
```diff
@@ -125,11 +125,23 @@ jobs:
       vmImage: '$(OSVmImage)'
 
     steps:
+      - powershell: |
+            Invoke-WebRequest -Uri "https://github.com/chidozieononiwu/azure-sdk-tools/releases/download/sdk-tools_v1.0/sdk-tools.zip" `
```
Comment (scbedd) [inline .azure-pipelines/client.yml]: Perfectly fine with this, but we gotta pull a release from the `Azure/azure-sdk-tools` repo if we want this to go into master. Can you send a PR for your script to azure-sdk-tools repo?

Comment (scbedd) [reply .azure-pipelines/client.yml]: And I'll work with you on how we could set up automatic github releases, etc. :)

Comment (bsiegel) [reply .azure-pipelines/client.yml]: Anywhere you use the `Invoke-WebRequest` action, you should supply the `-UseBasicParsing` argument. Without this, an error is thrown on some request and platform combinations.

Comment (bsiegel) [reply .azure-pipelines/client.yml]: Alternatively you could just `git clone` the repo :) I think we do this in a few other places

Comment (chidozieononiwu) [reply .azure-pipelines/client.yml]: > Alternatively you could just `git clone` the repo :) I think we do this in a few other places

Yh, but we didn't want to have to clone the whole repo.

Comment (bsiegel) [reply .azure-pipelines/client.yml]: > Yh, but we didn't want to have to clone the whole repo.

Yeah, if it's a big repo I'd be more concerned but with a smaller one it's not too bad. You can use `--depth 1` to do a shallow clone which further limits the data transfer. The nice thing is if you want to improve your script, you just merge the PR into master and there's no step 2 :)

Comment (weshaggard) [reply .azure-pipelines/client.yml]: Yes we considered doing a git clone but decided to try and github releases instead. I agree git clone isn't bad for small repo's but I do expect this repo to grow over time and I also expect that we will likely start depending on things that aren't simple text script files, I expect some compiled binaries to be used from the tools repo at some point so the github release seems to make the most sense, at least with what we know right now.

**Pull request merged**